src/api/payment-requests/payment-requests.md

```diff
@@ -467,12 +467,168 @@ them to find the Payment Request and proceed to pay.
 {% endjson %}
 
 
+### Pay a Payment Request by Id
+
+There are two methods of paying a payment request.
+The first uses Centrapay [Assets] and requires you to provide the Id of the asset to use for payment.
+Alternatively you can provide a transaction hash and the Centrapay [Asset Type] for any crypto payments that we support.
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/payment'
```

Comment:
Suggestion to update `/payment` to `/pay` to match the action/verb being performed by the user - thoughts?

Looks like `/pay` is used for our legacy routes too which is why it stood out to me

Reply:
The idea here is that a payment transaction is created when this endpoint is called. In the future, you might post multiple times to this endpoint to satisfy a payment request.  Happy to discuss though.

Reply:
This is updated to /pay and /refund